src/dev/build/tasks/create_cdn_assets_task.ts

```diff
@@ -38,17 +38,27 @@ export const CreateCdnAssets: Task = {
       const manifest = Jsonc.parse(readFileSync(path, 'utf8')) as any;
       if (manifest?.plugin?.id) {
         const pluginRoot = resolve(dirname(path));
+        const assetsSources = [
+          resolve(pluginRoot, 'public', 'assets'),
+          resolve(pluginRoot, 'assets'),
+        ];
+        // packages/core/apps/core-apps-server-internal/src/core_app.ts
```

Comment:
What is this comment intending to communicate?

Reply:
This is the source of where the paths come from... There is a bit of "weak" reference here because these paths are created by Kibana server at runtime and we are capturing them in code here.

I just followed the pattern in the rest of the file ([eg](https://github.com/elastic/kibana/blob/82f25ce1543cd872b85857c2a794d565526483fb/src/dev/build/tasks/create_cdn_assets_task.ts#L55)).

Ideally we could share these values by importing some consts or something, but I guess these paths really don't change often so this has been an "OK" way.

Reply:
I'm happy to open a follow up issue!